tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -584,27 +707,56 @@ def test_check_sfputil_low_power_mode(duthosts, enum_rand_one_per_hwsku_frontend
             logging.info("restoring {} physical interface {}".format(intf, phy_intf))
             new_lpmode = original_lpmode[intf].lower()
             lpmode_set_result = duthost.command("{} {} {}".format(cmd_sfp_set_lpmode, new_lpmode, intf))
-            assert lpmode_set_result["rc"] == 0, "'{} {} {}' failed".format(cmd_sfp_set_lpmode, new_lpmode, intf)
+            assert lpmode_set_result["rc"] == 0, (
+                "'{} {} {}' failed. "
+                "Please check the command output, DUT logs, and platform status for troubleshooting."
+            ).format(cmd_sfp_set_lpmode, new_lpmode, intf)
+
     time.sleep(10)
 
     logging.info("Check SFP lower power mode again after changing SFP lpmode")
     lpmode_show = duthost.command(cmd_sfp_show_lpmode)
     parsed_lpmode = parse_output(lpmode_show["stdout_lines"][2:])
     for intf in dev_conn:
         if intf not in xcvr_skip_list[duthost.hostname]:
-            assert intf in parsed_lpmode, "Interface is not in output of '{}'".format(cmd_sfp_show_lpmode)
-            assert parsed_lpmode[intf].lower() == original_lpmode[intf].lower(), \
-                "Unexpected SFP lpmode. actual:{}, expected:{}".format(
-                    parsed_lpmode[intf].lower(), original_lpmode[intf].lower())
+            assert intf in parsed_lpmode, (
+                "Interface '{}' is not in output of '{}'. "
+                "Please verify the SFP module is properly inserted and operational, "
```

Comment:
Just tell the failed reason, no need to tell user how to debug or fix it. Make the summary clean and clear.